notebooks/05_operationalize/aks_locust_load_test.ipynb

```diff
@@ -0,0 +1,392 @@
+{
```

Comment:
<p>This is difficult to see on Github, since the lines are breaking. Is there another way to show this? Maybe on a notebook?</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/758/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
hmm, that would be nice but would require a decent bit of text parsing which can get brittle. I tried a few different ways to print this cleanly without much success... could replace with a screenshot I suppose ?

Reply:
yeah, probably easier to leave it as it is than to have a screenshot. Resolving this